providers-sdk/v1/upstream/mvd/cvss/cvss.go

```diff
@@ -407,6 +407,11 @@ func MaxScore(cvsslist []*Cvss) (*Cvss, error) {
 
 	for i := 1; i < len(cvsslist); i++ {
 		entry := cvsslist[i]
+		if entry == nil {
```

Comment:
what is the use case for getting a nil cvss in that list? I think we should not put nils in the list